torch/fx/symbolic_trace.py

```diff
@@ -119,34 +113,31 @@ def is_leaf_module(self, m: torch.nn.Module, module_qualified_name : str) -> boo
         """
         return m.__module__.startswith('torch.nn') and not isinstance(m, torch.nn.Sequential)
 
+    def get_module_qualified_name(self, mod):
+        for n, p in self.root.named_modules():
+            if mod is p:
+                return n
+        raise NameError('module is not installed as a submodule')
+
     def call_module(self, m: torch.nn.Module, module_qualified_name: str, forward: Callable[..., Any], args, kwargs):
```

Comment:
once `get_module_qualified_name` is public then `call_module` doesn't need to take `module_qualified_name` as an argument, which would simplify things.